packages/cli/lib/lib/entry.js

```diff
@@ -48,7 +48,11 @@ if (typeof app === 'function') {
 		 * to send other data like at some point in time.
 		 */
 		const CLI_DATA = { preRenderData };
-		root = render(h(app, { CLI_DATA }), document.body, root);
+		const doRender =
+			process.env.NODE_ENV !== 'production' || root.tagName !== 'script'
```

Comment:
wondering if we need to recommend folks use `id="app"` here. We could have L35 do:

```js
let root = document.getElementById('preact_root') || document.body.firstElementChild;
```

Thoughts? It would help us avoid browser extensions potentially changing the value of firstElementChild on us.

Reply:
yes! I really really want to do this. This stream lines everything for us

Reply:
totally agree with it. :) Always wanted to do this one

Reply:
done